tests/testthat/test-backend-mssql.R

```diff
@@ -161,13 +161,12 @@ FROM `df`")
   )
 })
 
-test_that("ORDER BY in subqueries uses TOP 100 PERCENT (#175)", {
-  mf <- lazy_frame(x = 1:3, con = simulate_mssql())
-
-  expect_equal(
-    mf %>% mutate(x = -x) %>% arrange(x) %>% mutate(x = -x) %>% sql_render(),
-    sql("SELECT -`x` AS `x`\nFROM (SELECT TOP 100 PERCENT *\nFROM (SELECT TOP 100 PERCENT -`x` AS `x`\nFROM `df`) `dbplyr_001`\nORDER BY `x`) `dbplyr_002`")
-  )
+test_that("ORDER BY in subqueries uses TOP 9223372036854775807 (#337)", {
+  local_options(dbplyr_table_num = 0)
+  verify_output("sql/mssql-order-by.txt", {
+    mf <- lazy_frame(x = 1:3, con = simulate_mssql())
+    mf %>% mutate(x = -x) %>% arrange(x) %>% mutate(x = -x) %>% sql_render()
```

Comment:
Let's add another test, this will be fixed by #373.

Do lazy frames still need `sql_render()` in output tests?

```suggestion
    mf %>% mutate(x = -x) %>% arrange(x) %>% mutate(x = -x)

    mf2 <- lazy_frame(x = 3:1, con = simulate_mssql())
    mf %>% arrange(x) %>% left_join(mf2 %>% arrange(x))
```